include/fe/fe_abstract.h

```diff
@@ -244,35 +244,68 @@ class FEAbstract : public ReferenceCountedObject<FEAbstract>
   /**
    * \returns The \p xyz spatial locations of the quadrature
    * points on the element.
+   *
+   * It is overwritten by infinite elements since there
+   * \p FEMap cannot be used to compute \p xyz.
    */
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+  virtual
+#endif
   const std::vector<Point> & get_xyz() const
   { calculate_map = true; return this->_fe_map->get_xyz(); }
 
+
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+  /**
+   * This function is the variant of \p get_JxW() for \p InfFE.
+   * Since J diverges there, a respectize decay-function must be
```

Comment:
Typo: "respectize" -> may be a better word here is "suitable" or something along those lines.

Reply:
oh, I think I just meant respective :-) Thanks